File: 2016.3~2016.5/You're a Professional.py
F,I,T=map(int,input().split())
a=[0]*F
for i in range(F):
    a[i]=input()
b=0
for i in range(I):
    sum=0
    for j in range(F):
        sum+=(a[j][i]=='Y')
    b+=(sum>=T)
print(b)
#kitten

File: 2016.3~2016.5/You're a Professional.cpp
#include<stdio.h>
#define kitten

int main()
{
    int f, l, t;
    scanf("%d%d%d\n", &f, &l, &t);
    char fr[20][20];
    for (int i = 0; i < f; ++i) {
        scanf("%s", fr[i]);
    }
    int ans = 0;
    for (int i = 0; i < l; ++i) {
        int sum = 0;
        for (int j = 0; j < f; ++j) {
            if (fr[j][i] == 'Y') ++sum;
        }
        if (sum >= t) ++ans;
    }
    printf("%d\n", ans);
    return 0;
}
